code/_onclick/hud/fullscreen.dm

```diff
@@ -92,6 +92,13 @@
 		fs_view = client_view
 		transform = matrix(actualview[1]/FULLSCREEN_OVERLAY_RESOLUTION_X, 0, 0, 0, actualview[2]/FULLSCREEN_OVERLAY_RESOLUTION_Y, 0)
 
+/obj/screen/fullscreen/black
+	icon_state = "black" //just a black square, you can change this if you get better ideas
+	layer = FULLSCREEN_INTRO_LAYER
+
+/obj/screen/fullscreen/spawn
```

Comment:
Can you rename this typepath to something like spawnin so syntax highlighting doesnt break?